From 4af21cc90c2bd2ae0881069ad2f3c6bf02618cb3 Mon Sep 17 00:00:00 2001
From: Song Bing <b06498@freescale.com>
Date: Fri, 13 Mar 2015 18:04:31 +0800
Subject: [PATCH 05/12] encodebin: Need more buffers in output queue for better
 performance

Need more buffers in output queue for better performance

https://bugzilla.gnome.org/show_bug.cgi?id=744191

Upstream Status: Inappropriate [i.MX specific]

Signed-off-by: Song Bing <b06498@freescale.com>
Signed-off-by: Lyon Wang <lyon.wang@freescale.com>
---
 gst/encoding/gstencodebin.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/gst/encoding/gstencodebin.c b/gst/encoding/gstencodebin.c
index 55dab2f..4f20d68 100644
--- a/gst/encoding/gstencodebin.c
+++ b/gst/encoding/gstencodebin.c
@@ -1228,8 +1228,7 @@ _create_stream_group (GstEncodeBin * ebin, GstEncodingProfile * sprof,
    * We only use a 1buffer long queue here, the actual queueing will be done
    * in the input queue */
   last = sgroup->outqueue = gst_element_factory_make ("queue", NULL);
-  g_object_set (sgroup->outqueue, "max-size-buffers", (guint32) 1,
-      "max-size-bytes", (guint32) 0, "max-size-time", (guint64) 0,
+  g_object_set (sgroup->outqueue, "max-size-time", (guint64) 0,
       "silent", TRUE, NULL);
 
   gst_bin_add (GST_BIN (ebin), sgroup->outqueue);
-- 
1.9.1

